specification/cost-management/resource-manager/Microsoft.CostManagement/CostManagement/CostDetailsOperationResults.tsp

```diff
@@ -0,0 +1,69 @@
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+import "@typespec/openapi";
+import "@typespec/rest";
+import "./models.tsp";
+
+using TypeSpec.Rest;
+using Azure.ResourceManager;
+using TypeSpec.Http;
+using TypeSpec.OpenAPI;
+
+namespace Microsoft.CostManagement;
+// FIXME: CostDetailsOperationResults has no properties property
+/**
+ * The result of the long running operation for cost details Api.
+ */
+model CostDetailsOperationResults
```

Comment:
I think instead of a proxy resource or something, this looks more like this model: https://github.com/Azure/typespec-azure/blob/7007b8aca94e33643268e29289b4ade8c2ef7ab0/packages/typespec-azure-resource-manager/lib/responses.tsp#L143